.github/CONTRIBUTING.md

```diff
@@ -1,147 +1,89 @@
-# Contributing
+# Welcome to the Frontend Contributing Guide!
 
-HospitalRun is a community project. We invite your participation through
-financial contributions, issues, and pull requests!
+Before contributing, please read the [code of conduct](https://github.com/HospitalRun/hospitalrun/blob/master/.github/CODE_OF_CONDUCT.md).
 
-Contributions are always welcome. Before contributing please read the [code of conduct](https://github.com/HospitalRun/hospitalrun/blob/master/.github/CODE_OF_CONDUCT.md) and
-[search the issue tracker](https://github.com/HospitalRun/hospitalrun-frontend/issues); your issue
-may have already been discussed or fixed in `master`. If you're new to the project,
-maybe you'd like to open a pull request to address one of [good-first-issue](https://github.com/HospitalRun/hospitalrun-frontend/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
+## 1. Check out the [HospitalRun General Contributing Guide](https://github.com/HospitalRun/hospitalrun/blob/master/.github/CONTRIBUTING.md) for how to:
+   1. Browse Issues
+   2. Get Assigned an Issue
+   3. Fork the Repository and Create a Branch
+   4. Commit Changes
+   5. Submit a Pull Request
 
-Make sure to checkout the [HospitalRun Contributing Guide](https://github.com/HospitalRun/hospitalrun/blob/master/.github/CONTRIBUTING.md)
+## 2. After creating a local branch (step 3 above), follow these steps to configure CouchDB
 
-To contribute,
-[fork](https://help.github.com/articles/fork-a-repo/) this repo, commit your changes, and [send a Pull Request](https://help.github.com/articles/using-pull-requests/).
+CouchDB is the server side database which data from the frontend will sync to. CouchDB is required to login
+to HospitalRun. You could install and run CouchDB in any way you wish. For convienence, we have added a docker compose file in the
+root of this project to launch CouchDB. Below is the steps:
```

Comment:
"Below is the steps:" --> "Below are the steps:"

Reply:
"CouchDB is the server side database which data from the frontend will sync to" -> CouchDB is the server side database which data from the frontend will sync from and to.

Reply:
✅